Week1/homework-midy/exercise 1/index.js

```diff
@@ -0,0 +1,38 @@
+const books = [
+  {
+    title: 'The Design of Everyday Things',
+    author: 'Don Norman',
+    alreadyRead: false,
+  },
+  {
+    title: 'The Most Human Human',
+    author: 'Brian Christian',
+    alreadyRead: true,
+  },
+  {
+    title: 'You',
+    author: 'Caroline Kepnes',
+    alreadyRead: false
+  },
+  {
+    title: 'The Witcher',
+    author: 'Andrzej Sapkowski',
+    alreadyRead: true
+  },
+];
+
+let booksDiv = document.createElement('div');
+let bookList = document.createElement('ul');
+
+booksDiv.appendChild(bookList);
+
+for (let book in books) {
```

Comment:
It would be better to use for..of loop here. (books.forEach would be my personal preference)